python/vtool/data.py

```diff
@@ -4046,40 +3974,26 @@ def export_data(self, comment, selection=None):  # TODO: Refactor
                     nodes = model.get_nodes()
                     get_selection = False
 
-                selected_node_names = []
                 found = []
 
                 if get_selection:
                     selected_node_names = controller.get_graph().get_select_nodes()
-
-                    for node_name in selected_node_names:
-                        node_inst = controller.get_graph().find_node(node_name)
-                        if node_inst:
-                            found.append(node_inst)
-
-                if found:
-                    nodes += found
+                    found = [node_inst for node_inst in
```

Comment:
This should be changed, we are both overwriting and extending the declaration here.